### ISSUE TITLE
Semi-bold T inconsistent with regular T 

### ISSUE THREAD

**Issue by kauesena** [alerque/libertinus#195]
**EDIT:** What I originally referred to as bold is actually semi-bold. 

The semi-bold letter T isn't simply a heavier version of the normal T. It has extra strokes. In my opinion, it feels awkward compared to the normal T. That comparison happens in a natural way when the two characters are very close. That can happen when T appears in the title of a subsection formatted in semi-bold shape and the next paragraph begins with a capital T.
I've come to think that, probably, it was a design choice in order to add another level of distinction between T and **T**, but I think that the lateral strokes ending higher than the horizontal bar of **T** disrupts the uniformity of the font's capital letters and that the characters are already sufficiently different without this new level of distinction (It's possible to notice that by comparing of the semi-bold and the regular Th ligature.). 
![screenshot from 2018-07-08 19-55-18](https://user-images.githubusercontent.com/30255100/42424808-eb8890c8-82e8-11e8-9daf-d9579dafb164.png)

**Comment by fitojb** [comment]
I disagree that it’s wrong, it’s just the difference in height of the ascender of the _h_ and the cap height of the _T_. The ascenders are supposed to be higher than the cap height of uppercase letters (not taking into account any possible overshoot). This feature is typical of Garalde typefaces.

**Comment by kauesena** [comment]
Possibly, I haven't been clear enough about what I consider to be a problem. I'm sorry. I do not believe that there is something wrong with the Th ligature characters (neither in the bold character, nor in the regular one). I do not know the name of the strokes I believe to be unsuited, so I referred to them as the lateral strokes of the horizontal bar of **T**.  I believe I should have uploaded an image call attention precisely to what I meant. 
![tbars](https://user-images.githubusercontent.com/30255100/42460075-614de634-8373-11e8-94a1-5025767ca9b9.png)
I've just discovered that this character isn't actually bold, but semibold. I use the font with `unicode-math` and the package automatically chooses the semibold version for `\textbf`. Bellow there is a picture of the regular T, the semibold T and the bold T, respectively. 
![screenshot from 2018-07-09 12-25-02](https://user-images.githubusercontent.com/30255100/42460246-c3ec0b7c-8373-11e8-8f13-13920e3ccee7.png)
I believe that the semibold T (which I formerly believed to be bold T) should not have those lateral strokes that come over the horizontal bar of T.

**Comment by fitojb** [comment]
Ah, understood! FYI, those strokes are called serifs.

**Comment by kauesena** [comment]
Of course! How simple! 
Thanks!

**Comment by khaledhosny** [comment]
Feel free to send a pull request. Semibold seems to be a very neglected font with lots of outdated design decisions, though.

**Comment by waldyrious** [comment]
@alerque in the spirit of #333, this seems worth reopening. It appears to be a simple and straightforward fix.
